### PR TITLE
update claim doc

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: docs/claims.md [hunk 1]
```diff
@@ -16,30 +16,36 @@ The HATVaults contract only implements some time-out checks in case the arbitrat
...
    - `submitClaim` can only be called by the committee
    - `submitClaim` can only be called during a safety period
    - `submitClaim` can only be called if no other active claim exists
    - emergencyPause must not be in effect
 1. **CHALLENGE** `challengeClaim(_claimId)` 
-   - can only be called during the challengePeriod
+   - can only be called during the challenge period
    - only the `arbitrator` or the `registry.owner` can challenge a claim (the registry.owner functions here as a "challenger of last resort")
    - can only be called if the claim has not been challenged yet
-   - `_claimId` must be the id of the currently active claim
```
Comment (lirona) [inline docs/claims.md:29]: too obvious?

File: docs/claims.md [hunk 2]
```diff
@@ -16,30 +16,36 @@ The HATVaults contract only implements some time-out checks in case the arbitrat
...
 1. **APPROVAL** `approveClaim(_claimId, bountyPercentage)`
-   - during the challengePeriod:
-     - the claim must be challenged
-     - only the arbitrator can call `approveClaim` and can change the bountyPercentage if given the permission, until the challenge had timed out
-   - after the challenge period:
-     - if the claim was not challenged, anyone can call `approveClaim` and approve the claim. 
-     - the bountyPercentage remains that as chosen by the committee
-   - `_claimId` must be the id of the currently active claim
+   - during the challenge period and challenge timeout period:
```
Comment (lirona) [inline docs/claims.md:35]:
```suggestion
   - during the challenge period and challenge timeout period:
```

File: docs/claims.md [hunk 3]
```diff
@@ -16,30 +16,36 @@ The HATVaults contract only implements some time-out checks in case the arbitrat
...
+     - if the claim is challenged, and no more than `challengeTimoutPeriod` seconds have passed since the challenge, the arbitrator can call `approveClaim`. The arbitrator can change the bountyPercentage if given the permission
+   - during the challenge timeout period:
```
Comment (lirona) [inline docs/claims.md:37]:
```suggestion
   - during the challenge timeout period:
```

File: docs/claims.md [hunk 4]
```diff
@@ -16,30 +16,36 @@ The HATVaults contract only implements some time-out checks in case the arbitrat
...
+     - if the claim was not challenged, anyone can call `approveClaim` and approve the claim.  In that case, the bountyPercentage remains that as chosen by the committee
```
Comment (lirona) [inline docs/claims.md:38]:
```suggestion
    - if the challenge period had passed and the claim was not challenged (but is not yet expired), anyone can call `approveClaim` and approve the claim. In that case, the bountyPercentage remains that as chosen by the committee
```

File: docs/claims.md [hunk 5]
```diff
@@ -16,30 +16,36 @@ The HATVaults contract only implements some time-out checks in case the arbitrat
...
+      - anyone can dismiss the claim
+
+During the time from submitting a claim to its resolution (i.e. approval or dismissal), the vault will be locked for withdrawals, and no new claims can be submitted
+
+## Relevant settings
 
-During the time from submitting a claim to its resolution, the vault will be locked for withdrawals, and no new claims can be submitted
+- `challengePeriod`: length of challlenge period, in seconds
+- `challengeTimeoutPeriod`: length of time a challenged claim can be approved
+- `arbitratorCanChangeBounty`: determines whether the arbitrator change the bounty amount when approving a claim
```
Comment (lirona) [inline docs/claims.md:51]:
```suggestion
- `arbitratorCanChangeBounty`: determines whether the arbitrator can change the bounty percentage when approving a claim
```

File: docs/claims.md (hunk 1 above)
Comment (jellegerbrandy) [reply docs/claims.md:29]: read it with a stresss on "currently active"

**Pull request merged**